IMU_Testing/IMU_Testing.h

```diff
@@ -0,0 +1,43 @@
+#ifndef IMU_TESTING_H_
+#define IMU_TESTING_H_
+
+#include <Arduino.h>
+
+class IMU_Testing {
+  public:
+    IMU_Testing();
+    void setupEverything();
+    void setupOrient();
+    void setupGyro();
+    void setupAccel();
+    void printOrientData();
+    void printGyroData();
+    void printAccelData();
+    float getQX();
+    float getQY();
+    float getQZ();
+    float getRollRad();
+    float getPitchRad();
+    float getYawRad();
+    float getRollDeg();
+    float getPitchDeg();
+    float getYawDeg();
+    float getGyroX();
+    float getGyroY();
+    float getGyroZ();
+    float getAccelX();
+    float getAccelY();
+    float getAccelZ();
+    void checkOrient();
+    void checkGyro();
+    void checkAccel();
+    float calcAngleFromX(bool);
+    float calcAngleFromY(bool);
```

Comment:
The number of public methods seem a bit excessive. Try to trim this down. We only needs a absolute heading derive from accelerators and the heading from the DMP is all that should be available

Reply:
So do you want me to delete all the other functions other than getAccelX/Y/Z and calcAngleFromX/Y or do you want me to move them to private?